pkg/plugin/processor/builtin/registry.go

```diff
@@ -23,8 +23,13 @@ import (
 	"github.com/conduitio/conduit/pkg/foundation/cerrors"
 	"github.com/conduitio/conduit/pkg/foundation/log"
 	"github.com/conduitio/conduit/pkg/plugin"
+	"github.com/conduitio/conduit/pkg/plugin/processor/builtin/js"
 )
 
+var defaultBuiltinProcessors = map[string]ProcessorPluginConstructor{
```

Comment:
... expose this variable!
```suggestion
var DefaultBuiltinProcessors = map[string]ProcessorPluginConstructor{
```